packages/config/config/devices/0x032e/dat-101.json

```diff
@@ -0,0 +1,402 @@
+{
+	"manufacturer": "DEFARO",
```

Comment:
```suggestion
	"manufacturer": "Defaro",
```
or is the manufacturer name really written in uppercase?

Reply:
I just copied DAS-102 code

Reply:
Its likely that we didn't clean up that file yet - its a pretty manual process.